packages/wmr/demo/public/index.tsx

```diff
@@ -44,7 +44,7 @@ export function App() {
 }
 
 if (typeof window !== 'undefined') {
-	render(<App />, document.body);
+	hydrate(<App />, document.body);
```

Comment:
The `typeof window !== 'undefined'` browser check can be removed.

References:

https://github.com/preactjs/wmr/blob/fe51b7eb3a65740571863fbeebf3367b74a9ddfe/packages/preact-iso/hydrate.js#L7

https://github.com/preactjs/wmr/blob/main/packages/preact-iso/README.md#hydratejs
> This method also checks to make sure its running in a browser context before attempting any rendering - if not, it does nothing.